package.json

```diff
@@ -215,7 +215,7 @@
     "docs": "typedoc",
     "----------------------------------------- tests ---------------------------------------------------": "",
     "mocha": "mocha -r ./test/registerEsbuild",
-    "test-unit": "yarn mocha \"test/unit/**/*.test.[tj]s\"",
```

Comment:
if you want to run a single test suite, you can use `describe.only`. eslint will mark it as an error to stop you from committing it :)